pkg/services/login/authinfo.go

```diff
@@ -25,6 +25,8 @@ const (
 )
 
 func GetAuthProviderLabel(authModule string) string {
```

Comment:
@IevaVasiljeva  , @gamab or @kalleep perhaps you could answer as to why we do not have Okta in this list?

I was expecting it to be here 🤔